common/flatpak-dir.c

```diff
@@ -8171,7 +8233,7 @@ flatpak_dir_cleanup_undeployed_refs (FlatpakDir   *self,
     return FALSE;
 
   local_flatpak_refspecs = find_matching_refs (local_refspecs,
```

Comment:
I think `local_refspecs` here is the wrong kind of hash table. (Found by checking all the callers of `find_matching_refs()`.)